exercises/clock/canonical-data.json

```diff
@@ -213,7 +213,12 @@
           "minute": 1,
           "add": 3500,
           "expected": "11:21"
-        },
+        }
+      ]
+    },
+    {
+      "description": "Subtract minutes",
+      "cases": [
         {
           "description": "subtract minutes",
           "property": "add",
```

Comment:
Do `add` and `subtract` need to be separate? How about a single `adjust` property for both?